src/pkg/utils/sget.go

```diff
@@ -150,3 +152,48 @@ func Sget(ctx context.Context, image, key string, out io.Writer) error {
 
 	return err
 }
+
+// CosignVerifyBlob verifies the zarf.yaml.sig was signed with the key provided by the flag
+func CosignVerifyBlob(blobPath string, sigPath string, keyPath string) error {
+	ctx := context.TODO()
+	keyOptions := options.KeyOpts{KeyRef: keyPath}
+	certRef := ""
+	certEmail := ""
+	certIdentity := ""
+	certOidcIssuer := ""
+	certChain := ""
+	certGithubWorkflowTrigger := ""
+	certGithubWorkflowSha := ""
+	certGithubWorkflowName := ""
+	certGithubWorkflowRepository := ""
+	certGithubWorkflowRef := ""
+	enforceSCT := false
+
+	return verify.VerifyBlobCmd(ctx, keyOptions, certRef, certEmail, certIdentity, certOidcIssuer, certChain, sigPath, blobPath, certGithubWorkflowTrigger, certGithubWorkflowSha, certGithubWorkflowName, certGithubWorkflowRepository, certGithubWorkflowRef, enforceSCT)
+}
+
+// CosignSignBlob signs the provide binary and returns the signature
+func CosignSignBlob(blobPath string, outputSigPath string, keyPath string) ([]byte, error) {
```

Comment:
This does not support keys with passwords :eyes:
![image](https://user-images.githubusercontent.com/3977569/228991748-3c24ff55-1efe-461c-b171-2c0881baa7dc.png)